modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -35,6 +35,38 @@ Perform the migration only when an interruption in service is acceptable.
 $ oc get Network.config.openshift.io cluster -o yaml > cluster-openshift-sdn.yaml
 ----
 
+. Verify that the  `OVN_SDN_MIGRATION_TIMEOUT` environment variable is set and is equal to `0s` by running the following command:
++
+[source,bash]
+----
+#!/bin/bash
+
+if [ -n "$OVN_SDN_MIGRATION_TIMEOUT" ] && [ "$OVN_SDN_MIGRATION_TIMEOUT" = "0s" ]; then
+    unset OVN_SDN_MIGRATION_TIMEOUT
+fi
+
+#loops the timeout command of the script to repeatedly check the cluster Operators until all are available.
+
+co_timeout=${OVN_SDN_MIGRATION_TIMEOUT:-1200s}
+timeout "$co_timeout" bash <<EOT
+until
+  oc wait co --all --for='condition=AVAILABLE=True' --timeout=10s && \
+  oc wait co --all --for='condition=PROGRESSING=False' --timeout=10s && \
+  oc wait co --all --for='condition=DEGRADED=False' --timeout=10s;
+do
+  sleep 10
+  echo "Some ClusterOperators Degraded=False,Progressing=True,or Available=False";
+done
+EOT
+----
+
+. Remove the configuration from the Cluster Network Operator (CNO) configuration object by running the following command:
++
+[source,terminal]
+----
+oc patch Network.operator.openshift.io cluster --type='merge' /  --patch '{"spec":{"migration":null}}'
+----
+
 . To prepare all the nodes for the migration, set the `migration` field on the Cluster Network Operator configuration object by entering the following command:
```

Comment:
Because you introduced the CNO abbreviation in step 3, you can use it here.
```suggestion
. To prepare all the nodes for the migration, set the `migration` field on the CNO configuration object by running the following command:
```
I also changed "entering" to "running" for consistency, as that looks like the terminology mainly used in this procedure.

Reply:
I'll grab this while I'm here for sure. Thanks.